### ISSUE TITLE
what the result of this code

### ISSUE THREAD

**Issue by ShuchunXu** [GuanlinLee/REAT#1]
Hi
I ran this code, and all parameters adopted the default settings in the code, which should be consistent with the experiments in the paper. However, the experimental results show that the model performs very poorly on the test data,
`
train E079: 100% 110/110 [00:28<00:00,  3.85it/s, acc_adv=68.6410, acc_cl=91.4404, loss=0.5546]
`
`
test E079: 100% 79/79 [00:27<00:00,  2.85it/s, acc_adv=8.8800, acc_cl=68.0400, loss_adv=0.9494, loss_cl=0.9494]
`
but the model performs well on the training data, basically in line with the results in the paper. What is the reason for this?

**Comment by GuanlinLee** [comment]
Would you mind giving me information about your running environment? You can try pytorch=1.17.0, because there could be some inconsistency between different versions.

**Comment by ShuchunXu** [comment]
Sure, My environment as the fig below
![屏幕截图 2024-04-29 192052](https://github.com/GuanlinLee/REAT/assets/51018369/3ce78b03-6128-4055-a493-a83e6503997d)
But are you sure the cause is the different environment? I will try  pytorch=1.17.0
